integration/mdc-migration/verify-golden.ts

```diff
@@ -0,0 +1,190 @@
+import {promises as fsPromises, Stats} from 'fs';
+import path from 'path';
+
+// Expected usage:
+// verify-golden \
+//   <should_approve (true | false)> \
+//   <golden_directory (relative to cwd)> \
+//   <golden_directory (relative to workspace)> \
+//   ...<ignored_files (relative to cwd)>
+
+const CURRENT_WORKING_DIRECTORY = process.cwd();
+const SHOULD_APPROVE = Boolean(process.argv[2]);
+const TEST_DIRECTORY = CURRENT_WORKING_DIRECTORY;
+const GOLDEN_DIRECTORY = path.join(CURRENT_WORKING_DIRECTORY, process.argv[3]);
+const APPROVED_GOLDEN_DIRECTORY = path.join(
+  process.env.BUILD_WORKSPACE_DIRECTORY!,
```

Comment:
Nice!

Reply:
Note: CI is failing because the process variable is undefined in `bazel test`. Might want to do this later when approve is set to `true`

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:371:5)
    at validateString (node:internal/validators:119:11)
```